src/utils.ts

```diff
@@ -451,9 +452,32 @@ export function makeProfileJWT(profileData: Object, privateKey: string) : string
   const signedToken = blockstack.signProfileToken(profileData, privateKey);
   const wrappedToken = blockstack.wrapProfileToken(signedToken);
   const tokenRecords = [wrappedToken];
-  return JSONStringify(tokenRecords);
+  return JSON.stringify(tokenRecords);
```

Comment:
Please use `JSONStringify` here -- it will pretty-print the string if `stdout` isn't a TTY, and will print the same thing as `JSON.stringify()` otherwise.